test/metabase/test_runner/assert_exprs.clj

```diff
@@ -118,10 +118,11 @@
      :diffs    [[actual [only-in-expected only-in-actual]]]}))
 
 (defmethod t/assert-expr 'partial=
-  [message [_ expected & actuals]]
-  `(do ~@(for [actual actuals]
-           `(t/do-report
-             (partial=-report ~message ~expected ~actual)))))
+  [message [_ expected actual :as form]]
+  `(do
+     (assert (= 2 ~(count (rest form))) "partial= expects exactly 2 arguments")
+     (t/do-report
+       (partial=-report ~message ~expected ~actual))))
```

Comment:
IMO I would do the assertion during the macroexpansion itself rather than in the expanded form. Otherwise we wouldn't see the error until we actually tried to run the test. IMO it's better to catch these types of syntax errors as we read them in rather than waiting until the test is ran

```suggestion
  [message [_ expected actual :as form]]
  (assert (= (count (rest form)) 2) "partial= expects exactly 2 arguments")
  `(t/do-report
    (partial=-report ~message ~expected ~actual)))
```

Just my 2 cents tho. I can see an argument for doing it the other way too.

Either way `(= 2 (count ...))` is a bit of a [Yoda condition](https://en.wikipedia.org/wiki/Yoda_conditions) and I'd flip those around for readability